src/steps/10-FinancialDetails/IGCE/TravelEstimates.vue

```diff
@@ -1,15 +1,129 @@
 <template>
-  <div>
-    Future page for Travel Estimates
-  </div>
+  <v-container fluid class="container-max-width">
+    <v-row>
+      <v-col class="col-12">
+        <h1 class="page-header">
+          How do you want to estimate your travel needs?
+        </h1>
+        <p>
+          Consider the travel requirements that you previously outlined (e.g.,
+          location, duration, quantity, attendance, etc.). You may choose to set
+          a ceiling price and apply it to all periods. Or, if you know that your
+          travel requirements will change over time, then you may opt to set a
+          different price for each period.
+        </p>
+        <div class="copy-max-width">
+          <ATATRadioGroup
+            class="copy-max-width mb-10 max-width-740"
+            id="TravelEstimates"
+            :card="true"
+            :items="travelEstimateOptions"
+            @click="travelFormFields"
+            :value.sync="selectedTravelEstimate"
+            :rules="[$validators.required('Please select an option')]"
+          />
+        </div>
+
+      
+         <fieldset class="no-border" v-if="travelFormFields !==''">
+            <legend class="mb-4"><strong>Estimated travel costs per period</strong></legend>
+            <template v-if="travelFormFields === 'single'">
+              <ATATTextField
+                id="SingleAmount"
+                :value.sync="Amounts[0]"
+                :alignRight="true"
+                :isCurrency="true"
+                :showErrorMessages="true"
+                @blur="sanitizeValue(0, Amounts[0])"
+                width="190"
+                class="mr-2"
+                :rules="[$validators.required('', true)]"
+              />
+            </template>
+            <template v-if="travelFormFields === 'multiple'">
+              <div
+                v-for="(period, idx) in periods"
+                :key="idx"
+                :class="[(idx < periods.length-1 ? 'pb-5': ''), ' d-flex align-center']"
+                style="border-left: #544496 4px solid;"
+              >
+                <div class="text-right" style="width: 75px">
+                  {{ getOption(idx) }}
+                </div>
```

Comment:
when errored, the base/option field labels should stay inline with the input. They're shifting down.
![Screen Shot 2022-10-06 at 9 49 27 AM](https://user-images.githubusercontent.com/90333349/194330808-667e4171-df62-4624-9dd1-c84f97b775be.png)

Reply:
can you pls change `my-3` to `mt-2` ? it's sitting a little low now in relation to the input